dev/com.ibm.ws.springboot.fat20.concurrency.app/src/main/java/com/example/demo/AppRunner.java

```diff
@@ -0,0 +1,66 @@
+package com.example.demo;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+import java.util.concurrent.CompletableFuture;
+
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+@Component
+class AppRunner implements CommandLineRunner {
+
+	private final static Logger logger = LoggerFactory.getLogger(AppRunner.class);
+
+	private static ConcurrencyTasks concurrencyApplicationConfig = new ConcurrencyTasks();
+
+	public AppRunner(ConcurrencyTasks concurrencyApplicationConfig) {
+		this.concurrencyApplicationConfig = concurrencyApplicationConfig;
+	}
+
+	@Override
+	public void run(String... args) throws Exception {
+		try {		
+			assertManagedThread("AppRunner");
+		} catch (Exception e) {
+			logger.error("Exception on assertManagedThread.", e);
+		}
+	}
+
+	public static void assertManagedThread(String message) throws Exception {
+		//assert not null of the JNDIlookups for TransactionManager and DefaultManagedScheduledExecutorService
+		try {
+			InitialContext ic = new InitialContext();
+			Object tm = ic.lookup("java:comp/TransactionManager");
+			Object dmses = ic.lookup("java:comp/DefaultManagedScheduledExecutorService");
+			assertNotNull("Transaction manager lookup failed", tm);
+			assertNotNull("DefaultManagedScheduledExecutorService", dmses);
+		} catch (NamingException e) {
+			logger.error(message + ": JNDI LOOKUP FAILED", e);
+			fail("Transaction manager lookup failed: " + e.getMessage());
+		}
+		logger.info(message + ": MANAGED THREAD VERIFICATION PASSED");
+	}
+	
+	//Call a new static method assertAsyncMethod, it will call Task 1 and 2, takes a String message as arg
+	//Verifies the result of the CompletableFuture, assert on the string value returned by the tasks
+    //Log the message based on success on failure
+    public static void assertAsyncMethod(String message) throws Exception {
```

Comment:
This should not be a `static` method so it can access the non-static field `concurrencyApplicationConfig`.

That may make things circular in the one place `assertAsyncMethod` is called from `scheduledTask`.  It would probably be better to inject `concurrencyApplicationConfig` into `MyTask` instead and have the `assertAsyncMethod` logic there instead.